### ISSUE TITLE
initializing color hard to view

### ISSUE THREAD

**Issue by awick** [mobz/elasticsearch-head#178]
As a Red-Green Color Blind (http://en.wikipedia.org/wiki/Color_blindness) person the initializing and started colors look the same.  Would it be possible to change initializing to a different color, maybe just "yellow".

**Comment by jthigpen** [comment]
I ended up using the chrome extension "Stylinsh" to accomplish this same thing for the same reasons.

This is the style override I used.
```
.uiNodesView-replica.state-INITIALIZING { background-color: orange; }
```

**Comment by awick** [comment]
Good idea, right now I just edit app.css every time :)  Another one that is extremely useful is

    .uiNodesView-nullReplica {display: none;}

**Comment by brunocvcunha** [comment]
+1.

Can not tell the status of the cluster with the provided colors.

![image](https://cloud.githubusercontent.com/assets/3207647/18258731/20584baa-7390-11e6-8e80-13fc4e542140.png)

Deuteranopia (red/green color blindness) is common, and the fix should be too. You can pick different colors, but there are different levels, so a good way is linking states to image patterns.
